security/sec-iride/src/main/java/org/geoserver/security/iride/IrideUserGroupService.java

```diff
@@ -0,0 +1,183 @@
+/*
+ *  GeoServer Security Provider plugin with which doing authentication and authorization operations using CSI-Piemonte IRIDE Service.
```

Comment:
Usually header's comments contain copyright. Description of the class should go below, on the JavaDoc of the class so that it will be visible when using the APIs.